### PR TITLE
User Signing out - Client

### LINKED ISSUE #115: User Signing out - Client
## The story:

As a **signed-in user** that **wants to sign out**, I want to be able to **click on a button to sign out**.

---

## What we should do:
Build a sign-out button that logs the user out, redirects to the login page and return a message saying:

> You have signed out

---

## To be successful we need to:

- [ ] Make a _Sign Out_ button
- [ ] Redirect the signed-out user to the login page
- [ ] Return a message that the user has signed out
- [ ] Contain relevant tests for work done that pass in CI

---
#113 - Related to the API issue of signing out

### PR DESCRIPTION
**Pull request by wzhang66**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

## Fixed issues (PRs without this will not be accepted)
Fixes #115
- Add a Sign Out Button component allow the user to sign out
- Redirect the user to the login page
- Alert the user they have signed out by using console.log()

## Changes
- Addition of SignOut file in src/components with SignOutBtn.tsx and SignOutBtn.module.css files
- Displaying the button at the AppHeader component for functionality

## PR Checklist
- [ ] Does your PR contain relevant tests that pass?
- [ ] Is your code linted? `yarn lint` or `yarn lint:fix`
- [ ] Does your code build correctly in ALL projects? `yarn build`

### CODE REVIEW

File: packages/client/src/components/SignOutBtn/SignOutBtn.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+export const SignOutBtn = () => {
+  const { signOut } = Auth.useContainer();
+  return (
+    <button
+      className={classes.Btn}
+      type="button"
+      onClick={() => {
+        signOut();
+        console.log('You have signed out');
```
Comment (tristanMatthias) [inline packages/client/src/components/SignOutBtn/SignOutBtn.tsx:14]: Remove this please

File: packages/client/src/components/SignOutBtn/SignOutBtn.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import { Auth } from '../../containers/Auth.container';
+import classes from './SignOutBtn.module.css';
+
+export const SignOutBtn = () => {
+  const { signOut } = Auth.useContainer();
+  return (
+    <button
+      className={classes.Btn}
```
Comment (tristanMatthias) [inline packages/client/src/components/SignOutBtn/SignOutBtn.tsx:10]: Please use a standard button class for this.

File: packages/client/src/components/SignOutBtn/SignOutBtn.module.css [hunk 3]
```diff
@@ -0,0 +1,6 @@
+.Btn {
+    background-color: #3C60C5;
+    color: #FFF;
+    padding: 0.3rem;
+    border-radius: 0.3rem;
+}
```
Comment (tristanMatthias) [inline packages/client/src/components/SignOutBtn/SignOutBtn.module.css:6]: This can be deleted, as we should use a standard button class

**Comment by ghost** [comment]
Hi @wzhang66 

@tristanMatthias requested changes 12 days ago, but you have not responded.

Can you give an update on this PR?

**Comment by wzhang66** [comment]
@shaneSheridan I think I have already updated the PR and it is now waiting for the review?

**Pull request merged**